components/image-builder/pkg/builder/builder.go

```diff
@@ -44,7 +44,7 @@ const (
 	LabelTemporary = "gitpod.io/image-builder/temporary"
 
 	// maxBuildRuntime is the maximum time a build is allowed to take
-	maxBuildRuntime = 60 * time.Minute
+	maxBuildRuntime = 1 * time.Minute
```

Comment:
r u sure?

Reply:
See commit message(`DON'T MERGE, FOR TESTING ONLY`) and label (`do-not-merge/hold`) :upside_down_face: